vcd/metadata.go

```diff
@@ -2,18 +2,163 @@ package vcd
 
 import (
 	"fmt"
+	"github.com/hashicorp/go-cty/cty"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
 	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"github.com/vmware/go-vcloud-director/v2/util"
+	"regexp"
+	"strings"
 )
 
+var (
+	// IgnoreMetadataChangesConflictResolution can hold values "error", "warn" or "none", and tells Terraform whether it should
```

Comment:
I'd rather use `ConflictActions` instead of `ConflictResolution`.

Reply:
Thanks for spotting, I forgot to change all variable names when I changed the attribute. Now everything should be with the "Actions" suffix.